hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/dn/scanner/TestContainerScannerIntegrationAbstract.java

```diff
@@ -220,4 +220,8 @@ private OzoneOutputStream createKey(String keyName) throws Exception {
     return TestHelper.createKey(
         keyName, RATIS, ONE, 0, store, volumeName, bucketName);
   }
+
+  protected MiniOzoneCluster getMiniOzoneCluster() {
+    return cluster;
+  }
```

Comment:
We only need a getter for the one datanode in the cluster:
```suggestion
  protected HddsDatanodeService getDatanode() {
    assertEquals(1, cluster.getHddsDatanodes().size());
    return cluster.getHddsDatanodes().get(0);
  }
```